CHARTER.adoc

```diff
@@ -31,11 +36,17 @@ The TODO Group shall be composed of the General Members and a Steering Committee
  ... If any committee member should resign or otherwise be unable to complete their term for any reason, with more than six (6) months remaining in their term, the remaining committee members shall appoint a replacement member. For election or appointment purposes, that member's term will end when the outgoing member's term would have ended.
```

Comment:
```suggestion
 ... If any committee member should resign or otherwise be unable to complete their term for any reason, with more than six (6) months remaining in their term, the remaining committee members may appoint a replacement member. For election or appointment purposes, that member's term will end when the outgoing member's term would have ended.
```